tests/test_analytics_builders.py

```
@@ -0,0 +1,487 @@
+            cwi_deviation_within_20pct=80.0,
+            balance_trend="increasing",
+            budget_survival_pass_rate=85.0,
+            cwi_value=50.0,
+        )
+
+        dashboard_view = build_analytics_dashboard_view(
+            snapshot_orm=snapshot,
+            alerts_list=[],
+            events_list=[],
+            window_type="week",
+            window_start=datetime(2026, 8, 1),
```

Ruff (0.16.1)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)